README.md

```diff
@@ -40,6 +40,7 @@ Please read the instructions in the [contribution guide](CONTRIBUTING.md) in ord
 * The feedback system of [Frab][frab-github] and [Pretalx][pretalx-website]
 * [c3nav][c3nav-github] - An indoor navigation project
 * [Engelsystem][engelsystem-website] - Online tool for coordinating helpers and shifts on large events
+* [Chaosflix][chaosflix-github] - Android app for media.ccc.de, share Fahrplan-favorites with chaosflix to import them as bookmarks
```

Comment:
Please change `... Fahrplan-favorites with chaosflix ...` to `... Fahrplan favorites with Chaosflix ...`

Reply:
You removed the GitHub link accidentally. See https://github.com/NiciDieNase/EventFahrplan/tree/chaosflix_export (broken link). Also please remove the `-`.

Reply:
Resolved in cddf05260159ac640e19044158964a2f6be86536. Thanks.